jujugui/static/gui/src/app/app.js

```diff
@@ -2110,6 +2109,21 @@ YUI.add('juju-gui', function(Y) {
       next();
     },
 
+    _ensureControllerConnection: function(state, next) {
+      if (
+        state.root === 'logout' ||
+        this.controllerAPI.get('connecting') ||
+        this.controllerAPI.get('connected') ||
+        this.controllerAPI.userIsAuthenticated) {
+        console.log('ignoring controller connection');
```

Comment:
I'm not sure we want to log this as state changes quite frequently and this will be logged out every time.